tests/test_wave_05.py

```diff
@@ -71,9 +79,10 @@ def test_create_goal(client):
         }
     }
 
+
 @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    pass
+    raise Exception("Complete test")
```

Comment:
Do we want to be vague about how to complete the test?  I'm just uncertain about intent here.

Reply:
Since this is Wave 05, I thought it was reasonable not provide more of a scaffold. The readme suggest to use Wave 01 tests as a guide.

Reply:
Seems reasonable to me to provide less information with progressive waves and it feels like there's enough information in the project docs that they should know what needs to be completed.